### ISSUE TITLE
Add a module for Huawei HiSuite

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#9293]
### Is your feature request related to a problem? Please describe.
NVDA does not reads the Huawei HiSuite software.
### Describe the solution you'd like
An app module can be added for this program. NVDA's object review mode runs well, but normal navigation operations does not runs.
### Describe alternatives you've considered
If Huawei HiSuite uses an inaccessible interface, we can contact with Huawei.
### Additional context
[HiSuite download page](https://consumer.huawei.com/en/support/hisuite/)

**Comment by josephsl** [comment]
Hi, I think you meant to pour water into the cup in front of us, right? (i.e. please fill out the template if you can, thanks).

**Comment by zstanecic** [comment]
This issue is not filled against the template

**Comment by OzancanKaratas** [comment]
I have updated the issue comment.
